packages/cozy-harvest-lib/src/components/TriggerLauncher.jsx

```diff
@@ -141,9 +151,13 @@ export class TriggerLauncher extends Component {
         })}
         {showTwoFAModal && (
           <TwoFAModal
-            konnectorJob={konnectorJob}
             dismissAction={this.dismissTwoFAModal}
             into="coz-harvest-modal-place"
+            konnectorSlug={triggersModel.getKonnectorSlug(trigger)}
```

Comment:
I think the name triggersModel is deceptive since we do not have an instance of a model here. Maybe triggerLib ? triggers ?

Reply:
I used to use triggers 👍